client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue

```diff
@@ -37,6 +37,7 @@
       <dp-input
         id="videoTitle"
         v-model="video.title"
+        data-cy="uploadedVideoTitle"
```

Comment:
to relate this to the surrounding section hooks i'd change it to

```suggestion
        data-cy="customerSettings:videoTitle"
```